index.html

```diff
@@ -27,6 +27,8 @@
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
+        <!-- Adding some new improvments to navigation! -->
```

Comment:
Those improvements really provided a more streamlined approach. I would have gone with green instead of lime though. We may need to revisit the original request in the User Story.

Reply:
Consider using the `aria-current` on the navigation buttons to ensure that accessibility is addressed for visually impaired visitors. Something along the lines of:

```
<li>
	<a href="#main" aria-current="page">
	Space Bears
	</a>
</li>

```
This will help users know where they are after navigating through pages.